<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickPoints</class>
 <widget class="QDialog" name="PickPoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>55</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QComboBox" name="cbox_picktype">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../src.qrc">
          <normaloff>:/icon/resource/icon/edit-select.svg</normaloff>:/icon/resource/icon/edit-select.svg</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../src.qrc">
          <normaloff>:/icon/resource/icon/roll.svg</normaloff>:/icon/resource/icon/roll.svg</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_reverse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/reverse.svg</normaloff>:/icon/resource/icon/reverse.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/close-copy.svg</normaloff>:/icon/resource/icon/close-copy.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/add.svg</normaloff>:/icon/resource/icon/add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/remove.svg</normaloff>:/icon/resource/icon/remove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/start.svg</normaloff>:/icon/resource/icon/start.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../src.qrc">
         <normaloff>:/icon/resource/icon/reset.svg</normaloff>:/icon/resource/icon/reset.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections/>
</ui>
